src/Directory.Build.props

```diff
@@ -20,6 +19,7 @@
     <ManagedSourceCode>$(ProjectDir)Microsoft.Data.SqlClient\</ManagedSourceCode>
     <NetCoreSource>$(ManagedSourceCode)netcore\</NetCoreSource>
     <NetFxSource>$(ManagedSourceCode)netfx\</NetFxSource>
+    <CommonSource>$(ManagedSourceCode)src\</CommonSource>
```

Comment:
I see you added this. But I don't see it referenced anywhere. Did you intend to reference it in the csproj files or should it just be deleted?

Reply:
It can now be deleted, will remove it!